sdk/storage/Azure.Storage.Blobs/src/Models/BlobTagRequestConditions.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Azure.Storage.Blobs.Models
+{
+    /// <summary>
+    /// Specifies blob tag access conditions for a container or blob.
+    /// </summary>
+    public class BlobTagRequestConditions : RequestConditions
```

Comment:
I assume we want to rename this.

I added BlobTagRequestConditions because If-Tag applies to Blob Lease operations access conditions, which formally took a Request Conditions.  Blob Lease Operations access conditions do not include lease ID, which is why they never took BlobRequestConditions.

Reply:
I think I'd rather have `BlobLeaseRequestConditions` (named after entity APIs work on) in case this grows over time. and either repeat TagConditions in both `BlobLeaseRequestConditions` and `BlobRequestConditions` OR find some way to share that property.

Maybe
`BlobLeaseRequestConditions` < `Shared(or something)RequestConditions < `RequestConditions`
`BlobRequestConditions` < `Shared(or something)RequestConditions < `RequestConditions`

Reply:
I'll rename to `BlobLeaseRequestConditions`